### PR TITLE
[TRACKER-257] - Add Compliance Tab

### PR DESCRIPTION
**Pull request by sushil-car**
https://issues.cask.co/browse/TRACKER-257

### CODE REVIEW

File: cdap-ui/app/tracker/controllers/compliance-ctrl.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
```
Comment (awholegunch) [inline cdap-ui/app/tracker/controllers/compliance-ctrl.js]: 2017

File: cdap-ui/app/tracker/templates/compliance.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
+<!--
+  Copyright © 2016 Cask Data, Inc.
```
Comment (awholegunch) [inline cdap-ui/app/tracker/templates/compliance.html]: 2017

File: cdap-ui/app/tracker/templates/compliance.html [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+
+<div class="alert alert-danger error-msg" ng-show="ComplianceController.formattedData.length > 0">
+  <span>The following fields have failed compliance check based on <u ui-sref="tracker.dictionary">Data Dictionary</u></span>
```
Comment (awholegunch) [inline cdap-ui/app/tracker/templates/compliance.html]: These fields have failed a compliance check based on

Comment (elfenheart) [inline cdap-ui/app/tracker/templates/compliance.html]: don't use `<u>` as that element has been deprecated in HTML4

File: cdap-ui/app/tracker/templates/compliance.html [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+      </td>
+      <td>
+        <input type="checkbox" ng-checked="data.isPII" ng-disabled="true">
+      </td>
+      <td ng-class="{'not-exist': data.statusCode===404, 'not-matched': data.statusCode===409}">{{data.reason.join(', ')}}</td>
+    </tr>
+  </tbody>
+</table>
+
+<!-- <div class="preview-table-container" ng-hide="PreviewController.loading">
```
Comment (elfenheart) [inline cdap-ui/app/tracker/templates/compliance.html]: remove if not needed

File: cdap-ui/app/tracker/tracker.less [hunk 5]
```diff
@@ -907,6 +907,44 @@ body.state-tracker-detail-entity-preview {
...
+    }
+  }
+  .not-exist {
+    color: @tracker-green;
+  }
+  .not-matched {
+    color: #fa4347;
+  }
+  .error-msg {
+    u {
```
Comment (elfenheart) [inline cdap-ui/app/tracker/tracker.less]: still needed?

**Comment by elfenheart** [comment]
👍  LGTM. Please squash commits

**Comment by sushiladokar** [comment]
Squashing commit

**Pull request merged**